src/ark/utils/data_utils.py

```diff
@@ -792,3 +805,199 @@ def stitch_images_by_shape(data_dir, stitched_dir, img_sub_folder=None, channels
         current_img = stitched_data.loc['stitched_image', :, :, chan].values
         image_utils.save_image(os.path.join(stitched_subdir, chan + '_stitched' + file_ext),
                                current_img)
+
+
+@delayed
+def _convert_ct_fov_to_adata(fov_dd: dd.DataFrame, var_names: list[str], obs_names: list[str], save_dir=None) -> str:
```

Comment:
Will there ever be a case where `save_dir` is `None`? If so, it would throw an error for the following lines:

```
adata.write_zarr(pathlib.Path(save_dir, f"{fov_id}.zarr"), chunks=(1000, 1000))
return pathlib.Path(save_dir, f"{fov_id}.zarr").as_posix()
```

Reply:
`save_dir` can never be `None`. I'll remove that default value.